mesonbuild/dependencies/misc.py

```diff
@@ -125,7 +125,7 @@ def get_compile_args(self):
         # For now, use -isystem for all includes except for some
         # typical defaults (which don't need to be included at all
         # since they are in the default include paths)
-        if include_dir != '/usr/include' and include_dir != '/usr/local/include':
+        if not include_dir.endswith(('/usr/include', '/usr/local/include')):
```

Comment:
The fact this uses `endswith` is confusing when you first see it. Please add a comment saying this is done so it works with sysroot prefixes.